package.json

```diff
@@ -34,6 +34,7 @@
     "eslint-config-babel": "^6.0.0",
     "eslint-plugin-flowtype": "^2.20.0",
     "flow-bin": "^0.38.0",
+    "inspect-process": "^0.4.2",
```

Comment:
I don't know how others feel, but I usually see personal utils like this as good to install globally, since many people may test things in their own way.